linera-execution/src/evm/revm.rs

```diff
@@ -465,9 +465,12 @@ type Ctx<'a, Runtime> = MainnetContext<WrapDatabaseRef<&'a mut DatabaseRuntime<R
 // functionalities accessed from the EVM.
 const PRECOMPILE_ADDRESS: Address = address!("000000000000000000000000000000000000000b");
 
-// This is the zero address of the contract
+// This is the zero address used for service calls
 const ZERO_ADDRESS: Address = address!("0000000000000000000000000000000000000000");
 
+// This is the address being used for service calls
```

Comment:
The `ZERO_ADDRESS` above has the same comment.

Reply:
Ok, corrected.